kitchen-tests/.kitchen.travis.yml

```diff
@@ -2,7 +2,6 @@
 driver:
   name: dokken
   privileged: true
-  chef_image: chef/chef
```

Comment:
Old kitchen-dokken uses sean's image. The last pile of releases have defaulted to our images so this config becomes redundant.